lib/explorer/backend/series.ex

```diff
@@ -82,6 +82,7 @@ defmodule Explorer.Backend.Series do
   @callback less(s | valid_types(), s | valid_types()) :: s
   @callback less_equal(s | valid_types(), s | valid_types()) :: s
   @callback all_equal(s, s) :: boolean() | lazy_s()
+  @callback is_in(s, list() | s) :: s
```

Comment:
We convert to series in Explorer.Series, so the callback always receives series.

```suggestion
  @callback is_in(s, s) :: s
```